pdd/model.py

```diff
@@ -0,0 +1,70 @@
+import torch
+import torch.nn as nn
+from torchvision.models import mobilenet_v2
+import torch.nn.functional as F
+
+
+class Perceptron_classifier(nn.Module):
```

Comment:
Better name it `PerceptronClassifier` or choose one of `MLP`, `Perceptron`, etc. Class names usually presented without *snake_case* style of naming